src/tool/main.cpp

```diff
@@ -34,6 +34,8 @@
 
 namespace fs = std::filesystem;
 
+enum class result : int { ok = 0, error = 1 };
```

Comment:
I would personally use something better than ok, idk why it feels weird,
C standard adds exit_success and exit_failure macros.

I would write `enum class result : int { success = 0, failure = 1 };`

Reply:
But there was also nothing wrong with keeping an integer rather than enum class

Reply:
dont want magic numbers, prefer typed